arkouda/groupbyclass.py

```diff
@@ -248,7 +255,7 @@ def __init__(
         # This prevents non-bool values that can be evaluated to true (ie non-empty arrays)
```

Comment:
Just want to verify that this comment is still relevant since #1267 was closed.

Reply:
hmmm im not really sure, you added this comment in the PR that closed that issue
https://github.com/Bears-R-Us/arkouda/pull/1277/files

if a new dev looks at this and wonders why we are doing explicit type checking instead of using the `@typechecked`, this comment could lead them back to the issues that lead to this choice. I don't really think it's necessary. But ultimately i default to ur judgement since u added it

the full comment cause one line is missing from the diff
```
# Type Checks required because @typechecked was removed for causing other issues
# This prevents non-bool values that can be evaluated to true (ie non-empty arrays)
# from causing unexpected results. Experienced when forgetting to wrap multiple key arrays in [].
```

Reply:
Yeah I think its fine. Just wanted to get your take on it.